pkg/maps/nat/per_cluster_nat.go

```diff
@@ -196,8 +201,8 @@ func (om *PerClusterNATMap) getClusterNATMap(clusterID uint32) (*Map, error) {
 }
 
 func (om *PerClusterNATMap) cleanup() {
-	for i := uint32(1); i < perClusterNATMapMaxEntries; i++ {
-		om.deleteClusterNATMap(i)
+	for i := uint32(1); i <= cmtypes.ClusterIDMax; i++ {
+		_ = om.deleteClusterNATMap(i)
```

Comment:
Nit. It might be worth adding a comment about why this error is deliberately ignored. I'm also wondering whether it would be useful to emit a log message in case of errors, to simplify understanding possible failures.

Reply:
Added log message as well as missing `ClusterID` log field.